core/model/src/main/java/org/eclipse/sw360/antenna/api/workflow/ConfigurableWorkflowItem.java

```diff
@@ -76,6 +76,10 @@ public String getConfigValue(String key, Map<String, String> configMap, String d
         return result;
     }
 
+    public Boolean getBooleanConfigValue(String key, Map<String, String> configMap) throws AntennaConfigurationException {
```

Comment:
is this really necessary? it seems like it's just checking if people wrote "true" in uppercase letters.

Reply:
We have multiple cases where we check if a flag is set to `true` and all of these cases do the `"true".equals(..)` line. This start to move this duplicated code to a single place.